src/styles/blocks/_calendar.scss

```diff
@@ -0,0 +1,46 @@
+.calendar {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  width: ($cell-width + $cell-gap) * 7 + 20px;
+  padding: 10px;
+  flex-shrink: 0;
+  gap: $cell-gap;
+
+  @each $day, $number in $start-day {
+    &--start-#{$day} :first-child {
+      margin-left: ($cell-width + 1) * $number;
+    }
+  }
+
+  @for $index from 30 through 31 {
+    &--month-length-#{$index} {
+      .day:nth-child(n + #{$index + 1}) {
+        display: none;
+      }
+    }
+  }
```

Comment:
length modifier still not working, please double check the task description
![image](https://user-images.githubusercontent.com/80700247/230782574-5382fac0-5f07-403c-bea6-5d3aed0be3ad.png)